pkg/testcommon/testcommon.go

```diff
@@ -67,6 +69,23 @@ func (site *TestSite) Prepare() error {
 		site.Cleanup()
 	}
 
+	// If our test site has a Name: then update the config file to reflect that.
+	if site.Name != "" {
```

Comment:
Thanks for the idea. I hate to use nonstandard methods to access the config file, but that may be the ticket.